src/math.ts

```diff
@@ -57,12 +59,11 @@ export function rotate(
 
 export function adjustXYWithRotation(
   side: "n" | "s" | "w" | "e" | "nw" | "ne" | "sw" | "se",
-  position: { x: number; y: number },
+  element: ExcalidrawElement,
```

Comment:
@jquintozamora I'm hesitant to put `ExcalidrawElement` dependent code in `src/math.ts`. Let's keep pure math code here.
That said, it was misleading (of mine) to have `position` here. We could change it to `adjustXYWithRotation(side, x, y, deltaX, deltaY, angle)`.

Reply:
I see. I will revert back that change then :)